tutorials/01-mnist.ipynb

```diff
@@ -255,86 +384,42 @@
    },
    "outputs": [
     {
-     "data": {
-      "text/html": [
-       "<div>\n",
-       "<style scoped>\n",
-       "    .dataframe tbody tr th:only-of-type {\n",
-       "        vertical-align: middle;\n",
-       "    }\n",
-       "\n",
-       "    .dataframe tbody tr th {\n",
-       "        vertical-align: top;\n",
-       "    }\n",
-       "\n",
-       "    .dataframe thead th {\n",
-       "        text-align: right;\n",
-       "    }\n",
-       "</style>\n",
-       "<table border=\"1\" class=\"dataframe\">\n",
-       "  <thead>\n",
-       "    <tr style=\"text-align: right;\">\n",
-       "      <th></th>\n",
-       "      <th>mnistvid.data</th>\n",
-       "      <th>mnistimageclassifier.label</th>\n",
-       "    </tr>\n",
-       "  </thead>\n",
-       "  <tbody>\n",
-       "    <tr>\n",
-       "      <th>0</th>\n",
-       "      <td>[[[0, 0, 0], [0, 0, 0], [0, 0, 0], [0, 0, 0], ...</td>\n",
-       "      <td>6</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>1</th>\n",
-       "      <td>[[[2, 2, 2], [2, 2, 2], [2, 2, 2], [2, 2, 2], ...</td>\n",
-       "      <td>2</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>2</th>\n",
-       "      <td>[[[13, 13, 13], [2, 2, 2], [2, 2, 2], [13, 13,...</td>\n",
-       "      <td>3</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>3</th>\n",
-       "      <td>[[[0, 0, 0], [0, 0, 0], [0, 0, 0], [0, 0, 0], ...</td>\n",
-       "      <td>7</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>4</th>\n",
-       "      <td>[[[0, 0, 0], [0, 0, 0], [0, 0, 0], [0, 0, 0], ...</td>\n",
-       "      <td>5</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "</div>"
-      ],
-      "text/plain": [
-       "                                       mnistvid.data   \n",
-       "0  [[[0, 0, 0], [0, 0, 0], [0, 0, 0], [0, 0, 0], ...  \\\n",
-       "1  [[[2, 2, 2], [2, 2, 2], [2, 2, 2], [2, 2, 2], ...   \n",
-       "2  [[[13, 13, 13], [2, 2, 2], [2, 2, 2], [13, 13,...   \n",
-       "3  [[[0, 0, 0], [0, 0, 0], [0, 0, 0], [0, 0, 0], ...   \n",
-       "4  [[[0, 0, 0], [0, 0, 0], [0, 0, 0], [0, 0, 0], ...   \n",
-       "\n",
-       "  mnistimageclassifier.label  \n",
-       "0                          6  \n",
-       "1                          2  \n",
-       "2                          3  \n",
-       "3                          7  \n",
-       "4                          5  "
-      ]
-     },
-     "execution_count": 5,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "SELECT data, MnistImageClassifier(data) FROM MNISTVid WHERE ((id = 30) OR ((id = 50) OR ((id = 70) OR ((id = 0) OR (id = 140)))))\n"
```

Comment:
UDF is not created when using the new API.

Reply:
Given the current Pythonic API, which is serverless. I updated the notebook to directly use `EVACursor`, instead of using `connect`, since current `connect` still tries to look for eva server. We also need to call `init_builtin_udfs` explictly. Maybe #795 will improve the situation. @gaurav274 Please check the usage of Pythonic API in the notebook.

Reply:
@gaurav274 @xzdandy  With the auto-start server logic in #795, can we get rid of the need to call  `init_builtin_udfs` and just do `cursor = connect().cursor()` -- similar to https://dev.mysql.com/doc/connector-python/en/connector-python-api-mysqlcursor.html: 

```
import mysql.connector

cnx = mysql.connector.connect(database='world')
cursor = cnx.cursor()